src/include/uapi/ipc/topology.h

```diff
@@ -66,6 +66,12 @@ enum sof_comp_type {
 	SOF_COMP_FILEWRITE,	/**< host test based file IO */
 };
 
+enum sof_mux_flavour {
```

Comment:
I guess we need some SOF_COMP_DEFAULT_FLAVOR enum value that will be a default for other components.